scripts/replace-react-paths.js

```diff
@@ -0,0 +1,9 @@
+const gulp = require("gulp");
+const replace = require("gulp-replace");
+
+gulp.src('./dist/examples/react/**/*.html')
```

Comment:
just to be sure: https://github.com/voorhoede/fastatic/pull/54#discussion_r73964551 is not resolved by the changes.